src/Aspire.Cli/Projects/ProjectLocator.cs

```diff
@@ -23,7 +23,10 @@ private async Task<List<FileInfo>> FindAppHostProjectFilesAsync(DirectoryInfo se
         var appHostProjects = new List<FileInfo>();
 
         logger.LogDebug("Searching for project files in {SearchDirectory}", searchDirectory.FullName);
-        var projectFiles = searchDirectory.GetFiles("*.csproj", SearchOption.AllDirectories);
+        var enumerationOptions = new EnumerationOptions();
```

Comment:
I know there's a refactoring telling you to simplify this construction.

Reply:
I've saved the two characters, should run faster now ;)